iot/GAgent/package.json

```diff
@@ -0,0 +1,12 @@
+
+{
+    "name": "GAgent",
+    "description": "GAgent of Gizwits in RT-Thread",
+    "keywords": [
+        "GAgent"
+    ],
+    "readme": "GAgent of Gizwits in RT-Thread",
+    "site" : [
+    {"version" : "latest", "URL" : "https://github.com/qiaoqidui/ljson.git", "filename" : "ljson.zip", "VER_SHA": "master"}
```

Comment:
Please provide a correct git URL. And it's better to provide a fix and corrected version.